eng/pipelines/variables/esrp-signing-variables.yml

```diff
@@ -0,0 +1,17 @@
+#################################################################################
+# Licensed to the .NET Foundation under one or more agreements.                 #
+# The .NET Foundation licenses this file to you under the MIT license.          #
+# See the LICENSE file in the project root for more information.                #
+#################################################################################
+
+# These variables are used for running ESRP signing tasks. All names start with "Signing" to make
+# it clear that these variables are used for signing (as opposed to other msc tasks).
+
+variables:
+    - group: 'esrp-variables-v2'
```

Comment:
Can you just call it esrp-variables for now? If this ever needs updates in config in future, we can start with v2 later.

Reply:
Btw, if you're changing the group, do update all release branches/SNI branches next thing too - because we don't want to keep duplicate groups for such configs that serve all repos and branches with same configuration.

Reply:
I opted to call it -v2 because to me it is a second version of the signing variables. Since we've stored these variables in various, inconsistently named places, to me it makes more sense to make some indication that this group isn't one of the old ones.

As for changing all the other branches, yes, that is the goal. But, like I said at the pipeline meeting, I want the new pipelines (AKV provider, only so far) to reference the new tasks, variable groups, etc, as they are created. I'd like it to be entirely parallel to the existing ones since touching the existing ones is a lot riskier. But, I can send out a PR with those changes so we can expedite getting rid of the other group.

Reply:
Well, at the end of the day only this group will remain, so v2 doesn't really mean anything later. I'm saying this from a final view of what will remain after we cleanup (which we must asap) - this and many other groups that are contradictory with each other. There should only 1 group called 'esrp-variables-v2' (if you keep v2) that would contain the esrp stuff.